README.md

```diff
@@ -172,3 +172,8 @@ We recommend everyone start by working on
   is a good place to start learning the scikit-learn library as well as machine
   learning methodology and comes with lots of examples.
 - [This](https://builtin.com/data-science/supervised-machine-learning-classification) page has a nice overview of classification models.
+
+## Others
```

Comment:
I've already updated the environment filename and added seaborn on the master branch, so this section is not necessary in the README.

Reply:
Ok. I will go ahead and remove it.